test/docfx.Test/e2e/E2ESpec.cs

```diff
@@ -12,6 +12,8 @@ public class E2ESpec
 
         public readonly string Repo;
 
+        public readonly bool Only;
```

Comment:
>Only [](start = 29, length = 4)

can we just use mac VS/VSCODE to support that?

Reply:
vscode does not support it.